en/docs/build-and-run/tryit-tools/swagger-tool.md

```diff
@@ -0,0 +1,49 @@
+# Try HTTP services in Swagger view with Ballerina
+While you develop an HTTP service, you need to try it and debug it to check how it works. The Ballerina VS Code plugin provides the Swagger Try it view, which gives the ability to try HTTP services within VS Code instead of using any third-party software.
```

Comment:
```suggestion
While you develop an HTTP service, you need to try it and debug it to check how it works. The Ballerina VS Code extension provides the Swagger Try it view, which gives the ability to try HTTP services within VS Code instead of using any third-party software.
```